pkg/registry/registry/repository/storage/storage.go

```diff
@@ -119,6 +123,8 @@ type REST struct {
 	privilegedUsername string
 	harborClient       *harbor.APIClient
 	registryClient     *registryinternalclient.RegistryClient
```

Comment:
you should define distributionClient here same as harborClient , then use it to operate docker distribution

Reply:
done